requirements_dev.txt

```diff
@@ -5,4 +5,5 @@ gitchangelog
 mypy==1.3.0
 pre-commit==3.3.1
 pylint==2.17.4
-pytest==7.3.1
\ No newline at end of file
+pytest==7.3.1
+pandas
```

Comment:
please, pin it to a specific version; you can run `pip freeze` on your python environment to see which version you are running and for which the code works/tests passed